tools/devops/xcode-deps.csx.in

```diff
@@ -0,0 +1,23 @@
+#r "_provisionator/provisionator.dll"
+
+using System.IO;
+using System.Linq;
+
+using static Xamarin.Provisioning.ProvisioningScript;
+
+Exec ("defaults", "write", "com.apple.LaunchServices", "LSQuarantine", "-bool", "NO");
```

Comment:
that's a global setting

Reply:
This is the only part i'm not happy with, but we need to set the attrs somehow then

Reply:
you can do this to specific files using `xattr` like @dalexsoto mentioned it recently in #macios-tnt